fuzz/src/fuzz_targets/codec_number.rs

```diff
@@ -1,21 +1,24 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 #![no_main]
 #[macro_use]
 extern crate libfuzzer_sys;
 extern crate tikv;
+extern crate tikv_fuzz;
 
-use std::mem;
 use tikv::util::codec::number::*;
-
-fn make_u64<I>(iter: &mut I) -> u64
-where
-    I: Iterator<Item = u8>,
-{
-    let mut bytes = [0u8; 8];
-    for byte in &mut bytes {
-        *byte = iter.next().unwrap();
-    }
-    unsafe { mem::transmute(bytes) }
-}
+use tikv_fuzz::util::number_maker::{make_f64, make_i32, make_i64, make_u16, make_u32, make_u64};
```

Comment:
I think it's more usual to just import `number_maker` and use `number_maker::make_f64` and so on in the rest.

Reply:
got it~